packages/coreutils/src/time.ts

```diff
@@ -17,9 +17,7 @@ const UNITS: { name: Intl.RelativeTimeFormatUnit; milliseconds: number }[] = [
  * The namespace for date functions.
  */
 export namespace Time {
-  // Intl.RelativeTimeFormatStyle contains these, but it requires `ES2020.Intl`.
-  // We currently compile to an `ES2018` target.
-  export type HumanStyle = 'long' | 'short' | 'narrow';
+  export type HumanStyle = Intl.ResolvedRelativeTimeFormatOptions["style"];
```

Comment:
The `HumanStyle` type is no longer strictly necessary, but as it's exported, removing it would technically be a breaking change.